### PR TITLE
Some improvements for translatable texts

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
As discussed on Discord, https://discord.com/channels/818804595450445834/818852942882275379/1261271320180555787 ff.

### CODE REVIEW

File: src/notation/internal/notationuiactions.cpp [hunk 1]
```diff
@@ -1387,7 +1387,7 @@ const UiActionList NotationUiActions::m_actions = {
              mu::context::UiCtxNotationOpened,
              mu::context::CTX_NOTATION_OPENED,
              TranslatableString("action", "Othe&r…"),
-             TranslatableString("action", "Enter tuplet: create custom…")
+             TranslatableString("action", "Enter tuplet: other…")
```
Comment (Jojo-Schmitz) [inline src/notation/internal/notationuiactions.cpp:1390]: to match the actual string in the menu

File: src/notation/view/widgets/editstyle.ui [hunk 2]
```diff
@@ -11867,7 +11873,8 @@ text x-height):</string>
                       <string>Reset to default</string>
                      </property>
                      <property name="accessibleName">
-                      <string>Reset 'Dash Y position ratio' value</string>
+                      <string>
+                        Reset 'Vertical position (% of text x-height)' value</string>
```
Comment (Jojo-Schmitz) [inline src/notation/view/widgets/editstyle.ui:11877]: to match the actual string in the settings

File: src/notation/view/widgets/editstyle.ui [hunk 3]
```diff
@@ -11968,7 +11987,7 @@ first note of the system</string>
                       <string>Reset to default</string>
                      </property>
                      <property name="accessibleName">
-                      <string>Reset 'Melisma thickness' value</string>
+                      <string>Reset 'Underscore thickness' value</string>
```
Comment (Jojo-Schmitz) [inline src/notation/view/widgets/editstyle.ui:11990]: to match the actual string in the settings

File: src/notation/view/widgets/editstyle.ui [hunk 4]
```diff
@@ -11235,7 +11241,7 @@
                  <item row="3" column="4">
                   <widget class="QLabel" name="label_minDistance">
                    <property name="text">
-                    <string>Min. gap between lyrics:</string>
+                    <string>Min. gap between syllables/words:</string>
```
Comment (Jojo-Schmitz) [inline src/notation/view/widgets/editstyle.ui:11244]: this is about the horizontal distance. not between verses

File: src/notation/view/widgets/editstyle.ui [hunk 5]
```diff
@@ -11264,7 +11270,7 @@
                     </sizepolicy>
                    </property>
                    <property name="text">
-                    <string>Y position below:</string>
+                    <string>Position below:</string>
```
Comment (Jojo-Schmitz) [inline src/notation/view/widgets/editstyle.ui:11273]: The Y seems superfluous

File: src/notation/view/widgets/editstyle.ui [hunk 6]
```diff
@@ -11490,7 +11496,7 @@
                     </sizepolicy>
                    </property>
                    <property name="text">
-                    <string>Y position above:</string>
+                    <string>Position above:</string>
```
Comment (Jojo-Schmitz) [inline src/notation/view/widgets/editstyle.ui:11499]: here too

File: src/notation/view/widgets/editstyle.ui [hunk 7]
```diff
@@ -11694,7 +11700,7 @@
                       <string>Reset to default</string>
                      </property>
                      <property name="accessibleName">
-                      <string>Reset 'Dash pad' value</string>
+                      <string>Reset 'Dash gap' value</string>
```
Comment (Jojo-Schmitz) [inline src/notation/view/widgets/editstyle.ui:11703]: to match the actual string in the settings

File: src/notation/view/widgets/editstyle.ui [hunk 8]
```diff
@@ -12053,7 +12078,7 @@ first note of the system</string>
                       <string>Reset to default</string>
                      </property>
                      <property name="accessibleName">
-                      <string>Reset 'Melisma pad' value</string>
+                      <string>Reset 'Line gap' value</string>
```
Comment (Jojo-Schmitz) [inline src/notation/view/widgets/editstyle.ui:12081]: to match the actual string in the settings ("Melisma" and "gap" were wrong, and "Underscore" just got changed to "Extension line", here though "Line" is suficcient)

File: src/project/projectmodule.cpp [hunk 9]
```diff
@@ -137,13 +137,13 @@ void ProjectModule::resolveImports()
     auto er = ioc()->resolve<muse::extensions::IExtensionsExecPointsRegister>(moduleName());
     if (er) {
         er->reg(moduleName(), { EXEC_ONPOST_PROJECT_CREATED,
-                                TranslatableString("project", "On post project created") });
+                                TranslatableString::untranslatable("On post project created") });
```
Comment (Jojo-Schmitz) [inline src/project/projectmodule.cpp:140]: not used in the UI

File: src/project/projectmodule.cpp [hunk 10]
```diff
@@ -137,13 +137,13 @@ void ProjectModule::resolveImports()
...
         er->reg(moduleName(), { EXEC_ONPOST_PROJECT_OPENED,
-                                TranslatableString("project", "On post project opened") });
+                                TranslatableString::untranslatable("On post project opened") });
```
Comment (Jojo-Schmitz) [inline src/project/projectmodule.cpp:142]: same

File: src/project/projectmodule.cpp [hunk 11]
```diff
@@ -137,13 +137,13 @@ void ProjectModule::resolveImports()
...
         er->reg(moduleName(), { EXEC_ONPRE_PROJECT_SAVE,
-                                TranslatableString("project", "On pre project save") });
+                                TranslatableString::untranslatable("On pre project save") });
```
Comment (Jojo-Schmitz) [inline src/project/projectmodule.cpp:144]: same

File: src/project/projectmodule.cpp [hunk 12]
```diff
@@ -137,13 +137,13 @@ void ProjectModule::resolveImports()
...
         er->reg(moduleName(), { EXEC_ONPOST_PROJECT_SAVED,
-                                TranslatableString("project", "On post project saved") });
+                                TranslatableString::untranslatable("On post project saved") });
```
Comment (Jojo-Schmitz) [inline src/project/projectmodule.cpp:146]: same

File: src/engraving/types/typesconv.cpp [hunk 13]
```diff
@@ -243,7 +243,7 @@ static const std::vector<Item<ElementType> > ELEMENT_TYPES = {
     { ElementType::TEXTLINE_SEGMENT,     "TextLineSegment",      muse::TranslatableString("engraving", "Text line segment") },
     { ElementType::VOLTA_SEGMENT,        "VoltaSegment",         muse::TranslatableString("engraving", "Volta segment") },
     { ElementType::PEDAL_SEGMENT,        "PedalSegment",         muse::TranslatableString("engraving", "Pedal segment") },
-    { ElementType::LYRICSLINE_SEGMENT,   "LyricsLineSegment",    muse::TranslatableString("engraving", "Melisma line segment") },
```
Comment (Jojo-Schmitz) [inline src/engraving/types/typesconv.cpp:246]: "Melisma" was wrong

File: src/engraving/types/typesconv.cpp [hunk 14]
```diff
@@ -273,7 +273,7 @@ static const std::vector<Item<ElementType> > ELEMENT_TYPES = {
     { ElementType::TEXTLINE,             "TextLine",             muse::TranslatableString("engraving", "Text line") },
     { ElementType::TEXTLINE_BASE,        "TextLineBase",         muse::TranslatableString("engraving", "Text line base") },    // remove
     { ElementType::NOTELINE,             "NoteLine",             muse::TranslatableString("engraving", "Note line") },
-    { ElementType::LYRICSLINE,           "LyricsLine",           muse::TranslatableString("engraving", "Melisma line") },
+    { ElementType::LYRICSLINE,           "LyricsLine",           muse::TranslatableString("engraving", "Extension line") },
```
Comment (Jojo-Schmitz) [inline src/engraving/types/typesconv.cpp:276]: same

File: src/notation/internal/notationuiactions.cpp [hunk 15]
```diff
@@ -1716,8 +1716,8 @@ const UiActionList NotationUiActions::m_actions = {
     UiAction("add-melisma",
              mu::context::UiCtxNotationOpened,
              mu::context::CTX_NOTATION_TEXT_EDITING,
-             TranslatableString("action", "Add melisma"),
-             TranslatableString("action", "Lyrics: enter melisma")
+             TranslatableString("action", "Add extension line"),
+             TranslatableString("action", "Lyrics: enter extension line")
```
Comment (Jojo-Schmitz) [inline src/notation/internal/notationuiactions.cpp:1720]: "melisma" was wrong

File: src/notation/view/widgets/editstyle.ui [hunk 16]
```diff
@@ -7937,8 +7937,14 @@
                  </item>
                  <item>
                   <widget class="QToolButton" name="resetDynamicsAndHairpinPos">
+                   <property name="toolTip">
+                    <string>Reset to default</string>
+                   </property>
+                   <property name="accessibleName">
+                    <string>Reset 'Position' value</string>
+                   </property>
```
Comment (Jojo-Schmitz) [inline src/notation/view/widgets/editstyle.ui:7945]: missing

**Comment by Jojo-Schmitz** [comment]
@oktophonie, @bkunda mind to review this and prefer it over #23622?

File: src/notation/view/widgets/editstyle.ui (hunk 2 above)
Comment (cbjeukendrup) [inline src/notation/view/widgets/editstyle.ui:11877]: A newline has appeared here; I'm not sure whether that might have adverse effects

File: src/notation/view/widgets/editstyle.ui (hunk 4 above)
Comment (cbjeukendrup) [inline src/notation/view/widgets/editstyle.ui:11244]: This is the only one that we don't have a definitive agreement about internally, so still waiting for that

File: src/notation/view/widgets/editstyle.ui (hunk 2 above)
Comment (Jojo-Schmitz) [reply src/notation/view/widgets/editstyle.ui:11877]: OOps. Shouldn#t matter much, as this is for screen readers, but let's fix it anyhow

File: src/notation/view/widgets/editstyle.ui (hunk 4 above)
Comment (Jojo-Schmitz) [reply src/notation/view/widgets/editstyle.ui]: OK, I'll take it out of ths PR and put it into a separate one

Comment (Jojo-Schmitz) [reply src/notation/view/widgets/editstyle.ui]: See #23633

**Review by cbjeukendrup — APPROVED** [review]

**Pull request merged**